Automattic/pocket-casts-ios
# [Up Next Shuffle] Add Up Next Sheet

Adds the what's new for the UpNext Shuffle button


https://github.com/user-attachments/assets/a9a591f9-f5bb-440f-a61b-7910047df596



## To test

- CI must be 🟢 
- Add `Settings.lastWhatsNewShown = "7.72"` in the `AppDelegate` to reset the settings 
- Run the app
- You should see the new what's new view
- Dismiss it
- Stop Xcode
- Delete the line in the AppDelegate
- Rerun the app and confirm the view is not prompted anymore

## Checklist
